utensor_cgen/transformer/conv_pool.py

```diff
@@ -85,6 +87,91 @@ def transform(self, ugraph):
       update_tensor_op_names(ugraph)
       topologic_order_graph(ugraph)
       graph_validate(ugraph)
-    
-    #viz_graph('matcher', True, ugraph)
     return ugraph
+
+class ConvPoolTransformer(Transformer):
+  METHOD_NAME = 'conv_pool_v2'
+  KWARGS_NAMESCOPE = '_conv_pool_v2'
+
+  def __init__(self):
+    super(ConvPoolTransformer, self).__init__(prune_graph=False)
+
+  @property
+  def pattern_ugraph(self):
```

Comment:
Eventually it would be nice to figure out a better way of declaring the sub graphs. but the actual matching bit is so much nicer than it used to be

Reply:
Sure.
But it'll like doing our own `Tensorflow` python api again.
Not of first priority for now.

Reply:
Completely agree, let's just add this as a TODO feature in the issue tracker

Reply:
Right. Will do.

Reply:
https://github.com/uTensor/utensor_cgen/issues/80